src/eigen/lobpcg_hyper_impl.jl

```diff
@@ -431,4 +511,4 @@ end
     end
 
     final_retval(full_X, full_AX, resid_history, maxiter, n_matvec)
-end
+end
```

Comment:
We need newlines at the end of files, otherwise many tools panic. It's generally accepted practice I believe. I've setup my editor (emacs) to enforce it automatically long ago and never had trouble.

Reply:
I had never heard of this convention before, but after some reading it does make sense, and there is also an option in VSCode for it to do automatically. Thanks!